shell/browser/native_window_mac.mm

```diff
@@ -930,11 +933,18 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
 }
 
 void NativeWindowMac::SetMaximizable(bool maximizable) {
-  [[window_ standardWindowButton:NSWindowZoomButton] setEnabled:maximizable];
+  if (maximizable_ != maximizable) {
```

Comment:
Wouldn't the `SetMaximizable(maximizable_)` call in `RepositionTrafficLights` be no-op since `maximizable_ != maximizable` would always be false here?